source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -52,6 +58,17 @@ content: |
                key: localMasterKey,
              }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 2,3
+
+           kmsProviders = {
+             "local": {
+               "key": localMasterKey,
```

Comment:
Would it be better to use snake case to stay consistent with other variables like the first mention of `local_master_key`?